topology/m4/mixercontrol.m4

```diff
@@ -28,6 +28,79 @@ define(`CONTROLMIXER_OPS',
 `		put STR($4)'
 `	}')
 
+dnl KCONTROL_1CHANNELS(reg)
+define(`KCONTROL_1CHANNELS',
+`KCONTROL_CHANNEL(MONO, $1, 0)')
+
+dnl KCONTROL_2CHANNELS(reg)
+define(`KCONTROL_2CHANNELS',
+`KCONTROL_CHANNEL(FL, $1, 0),'
+`KCONTROL_CHANNEL(FR, $1, 1)')
+
+dnl KCONTROL_3CHANNELS(reg)
+define(`KCONTROL_3CHANNELS',
+`KCONTROL_CHANNEL(FL, $1, 0),'
+`KCONTROL_CHANNEL(FR, $1, 1),'
+`KCONTROL_CHANNEL(FC, $1, 2)')
+
+dnl KCONTROL_4CHANNELS(reg)
```

Comment:
We also need to watch the naming here, because it should also imply any channel mapping (rather than just calling it 4channels). Can you have a look and see if ALSA has a naming scheme for common channel mappings and reuse.

Reply:
Yes, I will